utilities/transfer-cards/main.go

```diff
@@ -43,7 +48,87 @@ func getProject(client *github.Client, ctx context.Context, projectName string)
 		}
 		opt.Page = resp.NextPage
 	}
-	return nil, fmt.Errorf("project '%s' not found in repo %s/%s", projectName, *repoOwner, *repoName)
+
+	// don't want to create a project if the project is a source project
+	if source {
+		return nil, fmt.Errorf("project '%s' not found in repo %s/%s", projectName, *repoOwner, *repoName)
+	}
+	// if this is a dry run do not create the project
+	if *dryrun {
+		return nil, fmt.Errorf("project '%s' not found in repo %s/%s if you would like the utility to create the project rerun without the dryrun option\n", projectName, *repoOwner, *repoName)
+	}
+
+	// if project was not found try creating the project
+	if getProjectErr != nil || project == nil {
+		log.Print("Project not found creating project\n")
+		project, createProjectErr := createProject(client, ctx, projectName)
+		if createProjectErr != nil {
+			return nil, createProjectErr
+		} else {
+			log.Print("Project creation successful\n")
+			return project, nil
+		}
+	} else {
+
+		return project, nil
+	}
+}
+
+// When a project is created the release bot needs to add project columns (triage, cherry-pick, and
+// cherry-picked) to the project. The transfer card utility should not progress until the release-bot is done
+func releaseBotDone(client *github.Client, ctx context.Context, projectID int) (bool, error) {
+	columnsLength := 0
+	retries := 1
+
+	for {
```

Comment:
instead of an infinite `for` loop here we should use something similar to a while loop:

```